### ISSUE TITLE
Failed test 'file_mtime_lt_ok success works'

### ISSUE THREAD

**Issue by eserte** [briandfoy/test-file#29]
In rare cases, the test suite may fail like this:
```
#   Failed test 'file_mtime_lt_ok success works'
#   at t/file_mtime.t line 48.
# STDOUT is:
#   ok 1 - file_mtime_lt_ok success
# > ok 2 - mtime_file mtime less than unix timestamp 1644348548
# not:
#   ok 1 - file_mtime_lt_ok success
# > ok 2 - mtime_file mtime less than unix timestamp 1644348547
# as expected
t/file_mtime.t ................ 
Failed 1/17 subtests 
```
Probably a race condition --- `time()` is called here three times, but there's no guarantee that the clock is returning the same epoch on every call. This can be easily demonstrated by adding an artitifical `sleep` like this:
```
--- a/t/file_mtime.t
+++ b/t/file_mtime.t
@@ -43,6 +43,7 @@ test_test( 'file_mime_age_ok failure works' );
 # file_mtime_lt_ok
 test_out( 'ok 1 - file_mtime_lt_ok success' );
 test_out( 'ok 2 - mtime_file mtime less than unix timestamp ' . time() );
+sleep 1;
 file_mtime_lt_ok( $mtime_file, time(), 'file_mtime_lt_ok success' );
 file_mtime_lt_ok( $mtime_file, time() );
 test_test( 'file_mtime_lt_ok success works' );
```

**Comment by briandfoy** [comment]
Yeah, not a good idea to call `time` three separate times there. Instead, I'll call it once. Note that the time in the expected error message needs to be the same as the time in the argument, though.

**Comment by briandfoy** [comment]
I think this is fixed in 1.992. If we run into again, we can re-open this issue.
